selfdrive/car/hyundai/interface.py

```diff
@@ -18,7 +18,12 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
 
     ret.carName = "hyundai"
     ret.safetyModel = car.CarParams.SafetyModel.hyundai
-    ret.radarOffCan = True
+    #not in cereal/car.capnp but I don't wanna fork cereal so
+    #ret.sccBus = 0 if 1056 in fingerprint[0] else 1 if 1056 in fingerprint[1] and 1296 not in fingerprint[1] \
+    #                                                                 else 2 if 1056 in fingerprint[2] else - 1
+    ret.radarOffCan = False if 1056 in fingerprint[0] else False if 1056 in fingerprint[1] and 1296 not in fingerprint[1] else False if 1056 in fingerprint[2] else True
```

Comment:
can you use the `is_ecu_disconnected` helper function here? (selfdrive/car/__init__.py)

Reply:
See https://github.com/commaai/openpilot/blob/master/selfdrive/car/toyota/interface.py#L246

Reply:
You can also only check on bus 0 for now.

Reply:
i'm assuming this would be without `or has_relay` because some vehicles might not have SCC, and if we do ever implement SCC emulation then radar data wouldn't be a part of that.

Reply:
resolved.